spec/features/editing_images/edit_image_modal_spec.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+RSpec.feature "Edit image in a modal", js: true do
```

Comment:
Hmm, that's an interesting idea. The behaviour does change in quite a few ways, though - the meta PR goes further with this. In this PR, there's one key difference: the `crop_x` ends up being `1` instead of `0`, which I assume is due to to the different dimensions of the modal.